### PR TITLE
Feature/cash 202

### PR DESCRIPTION
**Pull request by dubdabasoduba**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/openhmis/inventory/web/ModuleWebConstants.java [hunk 1]
```diff
@@ -69,5 +69,9 @@
 	public static final String LANDING_PAGE_EXTENSION_POINT_ID = "org.openmrs.module.openhmis.inventory.landing";
 	public static final String MANAGE_MODULE_PAGE_EXTENSION_POINT_ID = "org.openmrs.module.openhmis.inventory.manage.module";
 
+	public static final String MESSAGE_PROPERTIES_JS_URI = MODULE_ROOT + "inventoryMessageProperties.js";
+	public static final String MESSAGE_PAGE = MODULE_ROOT + "inventoryMessageProperties";
+	public static final String MESSAGE_PROPERTIES_JS = "/ws/module/openhmis/inventory/inventoryMessageProperties.js";
```
Comment (akmad) [inline api/src/main/java/org/openmrs/module/openhmis/inventory/web/ModuleWebConstants.java]: Why is this needed for the Inventory module but not for the Cashier module?

**Comment by akmad** [comment]
Looks good... just address the single question I have and then I will merge this in.

**Comment by akmad** [comment]
Also, make sure that there is an Inventory issue for this and let me know the ID. This should not be tagged under the cashier issue.

File: api/src/main/java/org/openmrs/module/openhmis/inventory/web/ModuleWebConstants.java (hunk 1 above)
Comment (dubdabasoduba) [inline api/src/main/java/org/openmrs/module/openhmis/inventory/web/ModuleWebConstants.java]: Because the localheader.jsp file in the inventory includes html pages using defined strings from the module web constants.
